.github/workflows/openapi-diff.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+on: [pull_request]
+jobs:
+  openapi-diff:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout HEAD
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+          path: head
+      - name: Checkout BASE
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.base_ref }}
+          path: base
+      - name: Generate oasdiff changelog
+        id: oasdif_changelog
+        run: | # Capture changelog as a multiline output
+          echo "changelog<<EOF" > $GITHUB_OUTPUT
+          docker run --rm \
```

Comment:
tufin/oasdiff does have a github action: https://github.com/Tufin/oasdiff-action

However, it can't run in "composed" mode, which compares globs of specs